neurons/validator.py

```diff
@@ -102,7 +102,4 @@ async def forward(self):
 
 # The main function parses the configuration and runs the validator.
 if __name__ == "__main__":
-    with Validator() as validator:
-        while not validator.should_exit:
-            bt.logging.info(f"Validator running... {time.time()}")
-            time.sleep(600)
+    Validator().run()
```

Comment:
`with Validator() as validator` will start a thread with target `run`. Let's call this run function directly

Reply:
that's fine, you decided not to run it in a separate thread because auto-restart didn't work that way?

Reply:
I looked at another subnet and they don't use threading. In our case, auto restart work but in the while loop we sleep 10 minutes, so it restart at worst 10 minutes after the crash.